tasties_app/views.py

```diff
@@ -61,3 +61,14 @@ def register(request):
 
     context = {'form': form}
     return render(request, 'tasties_app/register.html', context)
+
+
+@login_required(login_url='login')
+def single_recipe(request, recipe=None):
+    if not recipe:
+        recipe = Recipe.objects.first()
+    ingredients = Ingredient.objects.filter(recipe_id=recipe)
```

Comment:
You might not even have to do this. I think you can have: `ingredients = recipe.ingredient_set.all()` and get a slightly more readable assignment, maybe possibly but not sure even a slightly more efficient DB query.